static/js/components/SharePopup.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import React from "react"
+import onClickOutside from "react-onclickoutside"
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton
+} from "react-share"
+import { FacebookIcon, TwitterIcon, LinkedinIcon } from "react-share"
+
+export class SharePopupHelper extends React.Component<*> {
+  input: ?HTMLInputElement
```

Comment:
Should use `React.createRef()` instead: https://reactjs.org/docs/refs-and-the-dom.html#creating-refs

Reply:
nice, I didn't know about the new API